platform/app/.webpack/writePluginImportsFile.js

```diff
@@ -77,7 +73,7 @@ function getRuntimeLoadModesExtensions(modules) {
     if (module.importPath) {
       dynamicLoad.push(
         `  if( module==="${packageName}") {`,
-        `    const imported = await window.browserImportFunction('${isAbsolutePath(module.importPath) ? '' : publicURL}${module.importPath}');`,
+        `    const imported = await window.browserImportFunction('${publicURL}${module.importPath}');`,
```

Comment:
Please don't undo the change - if you want to prefix it, then use a relative path, and if you don't, then use an absolute path.  This change will break absolute paths to external modules.